src/sap.m/test/sap/m/qunit/SelectList.qunit.js

```diff
@@ -2031,6 +2031,36 @@ sap.ui.define([
 		oSelectList.destroy();
 	});
 
+	QUnit.test("with Delimited, it shouldn't prevent browser from navigating to previous or next page", function(assert) {
+
+		// system under test
+		var oSelectList = new SelectList({
+			keyboardNavigationMode: SelectListKeyboardNavigationMode.Delimited,
```

Comment:
This is the default value and we mention that in the description of test so remove this.